test/screenshot/lib/controller.js

```diff
@@ -238,6 +240,45 @@ class Controller {
       );
   }
 
+  /**
+   * Writes the given `testCases` to a `golden.json` file in `sourceDir_`.
+   * If the file already exists, it will be overwritten.
+   * @param {!Array<!UploadableTestCase>} testCases
+   * @return {!Promise<{goldenFilePath:string, goldenData:!Object}>}
+   */
+  async updateGoldenJson(testCases) {
+    const goldenData = {};
```

Comment:
This will destroy any current data in the golden.json file, correct? I want to make sure I understand.

Reply:
Yes! It will overwrite all existing data in golden.json.

In a future PR, I'm planning to be a little smarter about it, and only update URLs for screenshots that actually changed. That will make Git commits more meaningful and easier to review.

However, I need to create a "diffing" PR first 😀

Reply:
![sounds good](https://media3.giphy.com/media/j7aXTwkeq4mg8/giphy-downsized.gif)